allvars.h

```diff
@@ -520,7 +520,7 @@ extern struct particle_data
     pthread_spinlock_t SpinLock;
 #endif
 
-    float GravCost;     /*!< weight factor used for balancing the work-load */
+    int GravCost;     /*!< weight factor used for balancing the work-load */
```

Comment:
That's true. If that's what we want. 

I think it a slightly weird flag, actually - hydro is roughly 2/3 and gravity is 1/3 at early times, for one thing. But also, everything is really a tree walk, so balancing following gravity should be the same as balancing following gas. Unless it is really trying to compensate for the cost of the star formation or something at low redshift?